ui/src/app/utils/stores.tsx

```diff
@@ -240,7 +240,7 @@ export interface MatchParams {
   sparkConsolePath?: string;
   username?: string;
   usernameWithoutGsuiteDomain?: string;
-  wsid?: string;
+  terraName?: string;
```

Comment:
This is the primary change requiring most of the other changes in this file, including `:wsid` -> `:terraName` path matching changes